src/scripts/components/Wallet.js

```diff
@@ -29,26 +29,32 @@ class Wallet extends Component<Props> {
   render () {
     const balance = this.props.user.balances.PTI
     return (
-      <Card title="Wallet">
+      <Card
+        title="Wallet"
+        maxWidth
+        footer={
+          ACTIVATE_SECURE_WALLET &&
+          !this.props.isWalletSecured && (
+            <TextButton
+              data-test-id="secure-wallet"
+              onClick={this.secureWallet}
+              accent
+            >
+              Secure Wallet
```

Comment:
translation file!

Reply:
Let's delete it?
What you think, @eliawk ?